core/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -926,7 +926,26 @@ class DeltaSuite extends QueryTest
       )
     }
   }
-
+  test("get touched files for update, delete and merge") {
```

Comment:
nit: missing an empty line between tests.

Reply:
noted and added in